frontend/components/card/search-result/CardSearchResult.vue

```diff
@@ -27,6 +27,12 @@
     >
       <CardSearchResultUser :user="user" :isPrivate="isPrivate" />
     </div>
+    <div
```

Comment:
I'm going to revert the changes to search result as this should be a stand alone component rather than an option of search results.